### PR TITLE
CAMEL-19997: new camel-smb component

### PR DESCRIPTION
**Pull request by orpiske**
I'm working on a new component as part of CAMEL-19997. This is the very first iteration of it (only as consumer for now)  ... Kindly appreciate reviews and comments.

### CODE REVIEW

**Comment by orpiske** [comment]
Note: also comes with a test-infra module for that.

**Comment by orpiske** [comment]
Marking as draft again. There's a valid test error I need to fix when I get back.

**Comment by orpiske** [comment]
Looking OK now.

File: parent/pom.xml [hunk 1]
```diff
@@ -952,6 +953,11 @@
                 <artifactId>camel-chunk</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.camel</groupId>
+                <artifactId>camel-cifs</artifactId>
```
Comment (oscerd) [inline parent/pom.xml:958]: Is thius really needed?

**Review by oscerd — CHANGES_REQUESTED** [review]
I don't think camel-cifs is supposed to be listed

File: parent/pom.xml (hunk 1 above)
Comment (orpiske) [reply parent/pom.xml:958]: Man, that's not needed. That's very strange. That was the old name I started this component with. I'll erase my local cache and rebuild. Thanks for pointing it out!

File: components/camel-smb/src/main/resources/META-INF/services/org/apache/camel/component/smb [hunk 2]
```diff
@@ -0,0 +1 @@
+class=org.apache.camel.component.cifs.SmbComponent
```
Comment (orpiske) [inline components/camel-smb/src/main/resources/META-INF/services/org/apache/camel/component/smb:1]: This is also wrong (just noticed based on @oscerd's comment above)

**Comment by orpiske** [comment]
Hey @oscerd, any other comments here? Otherwise I'll merge ... it would be good to have this one for 4.2.

**Comment by oscerd** [comment]
For me it's +1

**Review by oscerd — APPROVED** [review]

**Comment by orpiske** [comment]
Thanks! Let's get this one in.

**Pull request merged**